pynitrokey/nk3/device.py

```diff
@@ -56,6 +58,8 @@ class Nitrokey3Device(Nitrokey3Base):
     """A Nitrokey 3 device running the firmware."""
 
     def __init__(self, device: CtapHidDevice) -> None:
+        from . import PID_NITROKEY3_DEVICE, VID_NITROKEY
+
```

Comment:
To discuss, whether we should hide imports or keep them global. It might matter later during code refactoring.

Reply:
You mean having them inside the function?  In this case it is required to avoid circular imports.

Reply:
I see. In that case perhaps constants should be in a separate module, without further references.
Let's keep this in mind.